packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -559,8 +559,16 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
         selectAll?: boolean
     ) {
         const range = editor.getDocument().createRange();
-        if (selectAll) {
-            range.selectNodeContents(cell);
+        if (selectAll && cell.firstChild) {
+            const cellStart = cell.firstChild;
+            const cellEnd = cell.lastChild || cell.firstChild;
```

Comment:
actually, if lastChild is null, firstChild must also be null. So just use lastChild should be fine